src/LandPlanner.cpp

```diff
@@ -24,26 +24,37 @@ LandPlanner::LandPlanner(
       transform_wrapper_(),
       state_(LandState::DONE),
       requested_height_(0.0),
+      cushion_height_(0.0),
       actual_descend_rate_(0.0),
       descend_rate_(ros_utils::ParamUtils::getParam<double>(
               private_nh,
               "descend_rate")),
+      cushion_rate_(ros_utils::ParamUtils::getParam<double>(
+              private_nh,
+              "cushion_rate")),
       descend_acceleration_(ros_utils::ParamUtils::getParam<double>(
               private_nh,
               "descend_acceleration")),
+      cushion_acceleration_(ros_utils::ParamUtils::getParam<double>(
+              private_nh,
+              "cushion_acceleration")),
       last_update_time_(),
       startup_timeout_(ros_utils::ParamUtils::getParam<double>(
               private_nh,
               "startup_timeout")),
       update_timeout_(ros_utils::ParamUtils::getParam<double>(
               private_nh,
-              "update_timeout")),
+              "update_timeout")),                    
       uav_arm_client_(nh.serviceClient<iarc7_msgs::Arm>("uav_arm"))
 {
     landing_detected_subscriber_ = nh.subscribe("landing_detected",
                                  100,
                                  &LandPlanner::processLandingDetectedMessage,
                                  this);
+    ROS_ASSERT_MSG((descend_rate_ <= 0) && 
```

Comment:
Can we make this four separate asserts?  Then we'll know immediately which one failed